src/App.vue

```diff
@@ -49,11 +50,114 @@ export default {
       // Message type (Info, Success, or Error) to display on banner
       messageType: 'Info',
       // API key from openweathermap.org - Unique to each person
-      openweathermapApiKey: '',
-      // Batul: ""
+      openweathermapApiKey: '5b59d0f6dbbe87dd97035b964b2b163b',
+
     }
   },
+  created() {
+    // Perform a check that the API key from openweathermap.org is defined
+    if (this.openweathermapApiKey === '') {
+      this.messageType = 'Error'
+      this.messageToDisplay = 'Error! API Key needs to be loaded to use openweathermap.org!'
+    }
+  },
+  methods: {
+    searchCity(inputCity) {
+      // GET request for user data
+      console.log(this.openweathermapApiKey);
+      axios.get('http://api.openweathermap.org/data/2.5/weather?q=' + inputCity + '&units=imperial&APPID=' + this.openweathermapApiKey)
+        .then((response) => {
+          // handle success
+          // this.messageType = 'Success'
+          // this.messageToDisplay = 'SUCCESS! Weather data was retrieved for ' + response.data.name + '!'
+          console.log(response, "response")
+
+          this.weatherData.city = response.data.name
+          this.weatherData.weatherSummary = response.data.weather[0].main
```

Comment:
is there any other way to access the 0 index data `response.data.weather[0]` ? we should not relay on indexes